src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -765,7 +770,7 @@ private async Task ConfigureVmAsync(ConnectionInfo sshConnectionInfo, IIdentity
 
             if (configuration.Update)
             {
-                await ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"sudo docker-compose down");
+                await ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"sudo docker-compose -f {CromwellAzureRootDirSymLink}/docker-compose.yml down");
```

Comment:
Update appears to have been broken in main, had to add this back in to make it work.